servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -125,7 +127,7 @@ def get_job(id, **kwargs):
         outputs=update_key_names(job.get('outputs', {})),
         labels=job.get('labels'),
         failures=failures,
-        extensions=ExtendedFields(tasks=sorted_tasks))
+        extensions=ExtendedFields(tasks=sorted_tasks, timing_url=timing_url))
```

Comment:
Where does the timing_url get displayed now?

Reply:
Same as before - it becomes a second link next to "List View" on the Tasks panel

Reply:
Oh, I see now! I didn't realize that link was still there

Reply:
Yeah, that's what I mean about seeming "too easy"... everything was already there except the URL!